pybop/optimisers/scipy_optimisers.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, method=None, bounds=None, maxiter=None, tol=1e-5):
         super().__init__()
         self.method = method
         self.bounds = bounds
+        self.num_resamples = 40
```

Comment:
This line is intentionally left without an set/get as #255 enables kwargs to this class, which will be added once that is merged.